content/docs/observability/_index.md

```diff
@@ -79,7 +79,7 @@ CSM for Observability provides Kubernetes administrators with the topology data
 | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Namespace                  | The namespace associated with the persistent volume claim                                                                                          |
 | Persistent Volume          | The name of the persistent volume                                                                                                                  |
-| Status                     | The status of the persistent volume. "Released" indicating the persistent volume has a claim. "Bound" indicating the persistent volume has a claim |
+| Status                     | The status of the persistent volume. "Released" indicating the persistent volume does not have a claim. "Bound" indicating the persistent volume has a claim |
```

Comment:
Line 82 should read "...indicates the persistent volume..." and "'Bound' indicates..."

Reply:
Thanks. Updated.